scripts/test-upgrade.sh

```
@@ -0,0 +1,136 @@
+jq '.app_state.ratelimit.hour_epoch.epoch_start_height = .initial_height' genesis.json | sponge genesis.json
+
+echo "ℹ️  Modifying gov voting periods..."
+jq '.app_state.gov.params.voting_period = "30s"' genesis.json | sponge genesis.json
+jq '.app_state.gov.params.expedited_voting_period = "20s"' genesis.json | sponge genesis.json
+
+mv genesis.json "$NODE_HOME"/config/genesis.json
+echo "✅   Genesis configured "
+
+echo "ℹ️  Configuring local validator..."
+echo "ℹ️  Adding local validator genesis account..."
+exrpd add-genesis-account $(exrpd keys show key -a --keyring-backend test --home "$NODE_HOME") 100000000poa,10000xrp --home "$NODE_HOME"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)